### PR TITLE
Directly reference the PodDisruptionBudget

### PR DESCRIPTION
**Pull request by stonean**
Save people from having to search for the translation of SLO to Service-Level Objectives.

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.14-ko.\<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/safely-drain-node.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ content_template: templates/task
 
 {{% capture overview %}}
 This page shows how to safely drain a machine, respecting the application-level
-disruption SLOs you have specified using PodDisruptionBudget.
+disruption SLOs (Service-Level Objectives) you have specified using PodDisruptionBudget.
```
Comment (mistyhacks) [inline content/en/docs/tasks/administer-cluster/safely-drain-node.md]: Don't capitalized the exploded acronym. Also, would it be better to remove the acronym and rephrase it to "respecting the PodDisruptionBudget you have defined"?

Comment (stonean) [reply content/en/docs/tasks/administer-cluster/safely-drain-node.md]: I knew that rule at some point, thank you for the reminder.

I prefer your rephrase suggestion. This wording extends the PodDisruptionBudget definition with this specific drain example. Since this page is linked from the "Disruptions" page in a "Learn more" context, I think it fits perfectly.

Comment (mistyhacks) [reply content/en/docs/tasks/administer-cluster/safely-drain-node.md]: I like it. Please make the change and ping me for re-review when it's done. Thanks!

**Comment by stonean** [comment]
@mistyhacks Changes made. There was a reference in the title and another toward the end of the post as well. Please let me know if these are acceptable.

I appreciate your help. Thank you!

**Comment by sftim** [comment]
@stonean you might want to tweak the PR title, as it's now out of sync with the commits.
(up to you!)

**Comment by mistyhacks** [comment]
/approve
/lgtm

Thanks for your contribution!
